packages/tailor-config-parser/src/schema.ts

```diff
@@ -45,8 +45,9 @@ const processElementConfig = (config: ElementConfig[]) => {
   }, [] as ContentElementCategory[]);
 };
 
-const processItem = (item: ContentElementItem | string) => {
-  return typeof item === 'string' ? { id: item as string } : item;
+const processItem = (item: ContentElementItem | string, config?: any) => {
```

Comment:
Name might be a bit too generic for contents of this script, consider increasing specificity